config/crd/kustomization.yaml

```diff
@@ -20,6 +20,9 @@ patchesStrategicMerge:
 # - patches/cainjection_in_clusteradmissionpolicies.yaml
 # - patches/cainjection_in_policyservers.yaml
 # - patches/cainjection_in_admissionpolicies.yaml
+#- patches/cainjection_in_admissionpolicies.yaml
+#- patches/cainjection_in_clusteradmissionpolicies.yaml
+#- patches/cainjection_in_policyservers.yaml
```

Comment:
This is duplicated

Reply:
fixed by @jvanz in https://github.com/kubewarden/kubewarden-controller/pull/247/commits/ff013d4ea058525b9f5f8dc156fb3541b058b9ac